app/controllers/admin/editions_controller.rb

```diff
@@ -338,6 +339,9 @@ def build_edition
     edition_locale = edition_params[:primary_locale] || I18n.default_locale
     I18n.with_locale(edition_locale) do
       @edition = LocalisedModel.new(new_edition, edition_locale)
+      if @edition.visual_editor.nil?
+        @edition.visual_editor = Flipflop.govspeak_visual_editor? && current_user.can_see_visual_editor_private_beta?
+      end
```

Comment:
I'm on the fence about whether this should be done on the controller. There's an [after_initialize](https://guides.rubyonrails.org/active_record_callbacks.html#after-initialize-and-after-find) callback we could use to do this automatically on the model. It does feel like more part of the domain model logic than the application logic. On the other hand Active Record models knowing about FlipFlop and the current user seems a bit grim, but that's sort of inevitable with AR.

Reply:
I tried a naive 
```
  def set_visual_editor_flag
    if visual_editor.nil? && created_at.nil?
      self.visual_editor = Flipflop.govspeak_visual_editor? && current_user.can_see_visual_editor_private_beta?
    end
  end
```
but it seems we do not have access to the current user in here.

Reply:
Yeah, fair enough. Seems the controller is the Railsy way